File: src/Front/ProEventos-App/src/app/models/Palestrante.ts
import { Evento } from "./Evento";
import { RedeSocial } from "./RedeSocial";

export interface Palestrante {

  id:Number;
  nome:string;
  minicurriculo:string;
  imagemURL:string;
  telefone:string;
  email:string;
  redeSociais:RedeSocial[];
  palestranteEventos:Evento[];

}

File: src/Back/src/ProEventos.Domain/Palestrante.cs
using ProEventos.Domain.Identity;

namespace ProEventos.Domain;

public class Palestrante
{
    public int Id { get; set; }
    public string Minicurriculo { get; set; }
    public int UserId { get; set; }
    public User User { get; set; }
    public IEnumerable<RedeSocial> RedeSociais { get; set; }
    public IEnumerable<PalestranteEvento> PalestranteEventos { get; set; }
}

